rebasehelper/options.py

```diff
@@ -179,6 +179,12 @@
         "switch": True,
         "help": "update \"sources\" file and upload new sources to lookaside cache",
     },
+    {
+        "name": ["--bugzilla-id"],
+        "metavar": "BUG_ID",
+        "default": None,
+        "help": "do a rebase based on Upstream Release Monitoring bugzilla"
+    },
```

Comment:
I think this option should be on top of `behavior control` list. Perhaps it would even deserve its own category.